types/types.go

```diff
@@ -71,7 +74,17 @@ type DockerImageManifest struct {
 	Labels        map[string]string
 	Architecture  string
 	Os            string
-	Layers        []string
+	FSLayers      []string
+}
+
+// Raw returns the raw manifest as received from the registry
+func (m *DockerImageManifest) Raw() []byte {
+	return m.RawManifest
+}
+
+// Layers returns a list of the FS layers in the manifest
+func (m *DockerImageManifest) Layers() []string {
+	return m.FSLayers
 }
```

Comment:
I'm thinking that OCI manifest for instance will have different members (or a completely new struct) so by having an interface (those methods satisfy the ImageManifest interface above) we could support other manifest struct